### PR TITLE
Upgrade H2 to 2.2.220

### PR DESCRIPTION
**Pull request by gsmet**
Created as draft to get a full CI run but `integration-tests/jpa-h2` passes with this patch.

### CODE REVIEW

**Comment by Sanne** [comment]
good idea, but previous H2 upgrades have been a disaster so let's check also ORM compatibility:
 - https://github.com/hibernate/hibernate-orm/pull/7162

**Comment by gsmet** [comment]
@sanne it's looking good for ORM. I will merge this as I want this in 3.2.4 for the Keycloak team (CVE).

**Comment by gsmet** [comment]
Thanks for testing btw!

**Comment by Sanne** [comment]
great, I'll update ORM for real in that case.

**Comment by gsmet** [comment]
Thanks for testing!

File: extensions/jdbc/jdbc-h2/runtime/src/main/java/io/quarkus/jdbc/h2/runtime/graalvm/DeleteFullTextLucene.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package io.quarkus.jdbc.h2.runtime.graalvm;
+
+import com.oracle.svm.core.annotate.Delete;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@Delete
+@TargetClass(className = "org.h2.fulltext.FullTextLucene")
+public final class DeleteFullTextLucene {
```
[the remark this answers is not in the document]
Comment (zakkak) [reply extensions/jdbc/jdbc-h2/runtime/src/main/java/io/quarkus/jdbc/h2/runtime/graalvm/DeleteFullTextLucene.java:8]: @gsmet this is a kind reminder

[the remark this answers is not in the document]
Comment (gsmet) [reply extensions/jdbc/jdbc-h2/runtime/src/main/java/io/quarkus/jdbc/h2/runtime/graalvm/DeleteFullTextLucene.java:8]: IIRC, it was an optional dependency and it was causing issues (we have a `quarkus-lucene` extension in the Quarkiverse but we don't want to drag it into the core).

**Pull request merged**